hvplot/tests/testcharts.py

```diff
@@ -347,6 +347,11 @@ def test_errorbars_no_hover(self):
         bkplot = Store.renderers['bokeh'].get_plot(plot)
         assert not bkplot.tools
 
+    def test_labels_format(self):
+        plot = self.df.hvplot("x", "y", text="({x}, {y})", kind="labels")
+        print(plot)
```

Comment:
Is this still WIP?

Reply:
Heh I was testing and I forgot to uncomment.

Reply:
Fixed now.